[error] java.io.FileNotFoundException: build.properties (Не удается найти указанный файл)
[error] 	at java.base/java.io.FileInputStream.open0(Native Method)
[error] 	at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
[error] 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
[error] 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)
[error] 	at service.MassageProcessingSevice$.runApp(MassageProcessingSevice.scala:18)
[error] 	at service.runApp.main(MassageProcessingSevice.scala:15)
[error] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[error] 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[error] 	at sbt.Run.invokeMain(Run.scala:144)
[error] 	at sbt.Run.execute$1(Run.scala:94)
[error] 	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:121)
[error] 	at sbt.Run$.executeSuccess(Run.scala:187)
[error] 	at sbt.Run.runWithLoader(Run.scala:121)
[error] 	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:2030)
[error] 	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1969)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at scala.util.Try$.apply(Try.scala:213)
[error] 	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:367)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
[error] 	at java.base/java.lang.Thread.run(Thread.java:1583)
[error] (Compile / run) java.io.FileNotFoundException: build.properties (Не удается найти указанный файл)
